### DISCUSSION TITLE
Very Slow Performance AMD Ryzen 5950X and Sabrent Rocket Raid 2TB Temp Drive

### DISCUSSION THREAD

**Discussion by neocleous** [swar/Swar-Chia-Plot-Manager#418] [Q&A]
Hi all,

I am testing Swar on my Linux plotting machine and I am getting very poor performance on it. 
![Swar](https://user-images.githubusercontent.com/47337157/118393257-131ba480-b636-11eb-82c5-2f3703d75bcd.PNG)
Using the GUI I can plot 7 at once with the destination drive being unique for each plot and I can do this three times in a day so this performance is well under par for this machine. This is the cinfig I am using.

name: 2TBSabrent
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: /media/neocleous/Sabrent1/Temp Plot
    temporary2_directory:
    destination_directory: /media/neocleous/Farm 1/Farm Plots
    size: 32
    bitfield: true
    threads: 2
    buckets: 128
    memory_buffer: 3390
    max_concurrent: 9
    max_concurrent_with_start_early: 9
    stagger_minutes: 45
    max_for_phase_1: 4
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 30
    temporary2_destination_sync: false

I have another machine running Windows 10 using a 5800X and a Samsung 2TB 980Pro and I am getting very good performance on that  using Swar and the config is very similar to the above so I am very confused.

**Comment by GSXswe** [comment]
https://www.youtube.com/watch?v=lv2avfUiBKk

The drive is much slower than a 980 Pro, thats why...  Watch the Youtube clip and you will understand.

But you can also try with more threads, go from 2 > 4 and see if it speeds up or not if you only run that drive for plotting in your system.

**Comment by neocleous** [comment]
Thanks for the link that was a very good video. I've reduced the length of my pipeline to see if that increases throughput. It's interesting that I seem to get more plots by doing 6 at once and going through the phases simultaneously rather than sequentially.

**Reply by GSXswe** [reply]
No problem  :-)   Oh, that is strange...  Phase 1 should be the more IO intensive one?
